.idea/libraries/stdlib_10_1_5.xml

```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```

Comment:
Those are usually not added, garbage like the rest of the .idea dir
part of the gitignore